### ISSUE TITLE
[Bug] Test fail upon erroneous Google Play data

### ISSUE THREAD

**Issue by raphaelvarieras** [fivetran/dbt_google_play#8]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

Recently, we noticed that some of our Google Play data contained errors at the source and the tests from the package are failing as a result.

Specifically, for the month of February, our source data contains two sets of ratings for the country of Curacao (CW). This causes the error below in our dbt pipelines.

### Relevant error log or model output

```shell
08:20:55  Failure in test dbt_utils_unique_combination_of_columns_stg_google_play__stats_ratings_country_package_name__date_day__country (models/stg_google_play.yml)
08:20:55    Got 28 results, configured to fail if != 0
```
```


### Expected behavior

If not unreasonable, I would expect the test to run on some base model in which the possibility of duplicate records has been addressed.

### dbt Project configurations

N/A

### Package versions

```
  - package: fivetran/google_play # https://github.com/fivetran/dbt_google_play
    version: 0.2.0  

  - package: fivetran/google_play_source # https://github.com/fivetran/dbt_google_play_source
    version: 0.2.0
```

### What database are you using dbt with?

snowflake

### dbt Version

1.4

### Additional Context

_No response_

### Are you willing to open a PR to help address this issue?

- [ ] Yes.
- [ ] Yes, but I will need assistance and will schedule time during our [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance
- [X] No.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @raphaelvarieras thanks for opening this issue!

Do you know why the duplicate data was introduced into your source data? Additionally, the tests are applied to the staging model as opposed to directly from the source. The tests are intended to highlight the duplicates as that could cause incorrect data in the downstream models of the package.

As such, we likely will not adjust this within the package as we will want to raise this to users if there are duplicates. If these duplicates in the source are a non issue for you, you can disable them with the following config to your dbt_project.yml:
```yml
tests:
  google_play_source:
    dbt_utils_unique_combination_of_columns_stg_google_play__stats_ratings_country_package_name__date_day__country:
      +enabled: false
```

If you feel this duplicate data should not exists, I would highly recommend opening a [support ticket](https://support.fivetran.com/hc/en-us) so our connector team can help investigate the duplicates and why they were introduced.

**Comment by raphaelvarieras** [comment]
Hey @fivetran-joemarkiewicz - I am afraid the duplicates actually should exist. I have verified it in the Google Play console that they are present in the source of the data. I have opened a ticket with Google about this. We will have to see if it gets anywhere.

In the short term, would your suggestion be that I just manually modify my source data (the landing table for Fivetran) and write down somewhere what I did in case we ever perform a full resync in the future?

**Comment by fivetran-joemarkiewicz** [comment]
Hey @raphaelvarieras thanks for the call out and additional context! 

Would you be able to share an example of these duplicate? Also, from the business use case do these duplicates make sense? If they do make sense, then we should adjust the package test. If they don't then I would be hesitant to adjust the package and would prefer to await Google's response to understand why these duplicates exist.

In the short term I would suggest using the configuration I shared in the above comment to temporarily disable the test.

**Comment by raphaelvarieras** [comment]
Sure - please see the content of the csv file straight out of the Google Play console:
<img width="497" alt="image" src="https://user-images.githubusercontent.com/6558178/225894424-a58a8f9e-9264-46bf-abc6-53cfb7a310e5.png">

I'll keep you posted on what Google says about this.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @raphaelvarieras I hope all is well. I just wanted to check in to see if you heard from Google if these duplicate records in the source is expected?

**Comment by raphaelvarieras** [comment]
No, still no answer from Google. I've ben told to "rest assured that we are actively asking for updates and we've already escalated this request to expedite the investigation. As of now, our engineering team is still looking into the issue."

**Comment by fivetran-joemarkiewicz** [comment]
Thanks for the update @raphaelvarieras! As this is still being investigated I will keep this issue open and recommend leveraging the test config shared in my first response to disable the test for the time being. 

However, please let us know if Google gets back to you and shares that this is in fact something our package should handle by default. In the interim I will mark this issue and wont fix, with the intention of either closing or accepting to update following news from Google regarding the data.

If any other users are experiencing this issue, please comment in this thread and we can do what we can to try and get more clear answers from Google. Thanks!

**Comment by raphaelvarieras** [comment]
Final response:

> As per our engineering team. The reason why the report generates two different data for curacao is because in our 
> database, there are two country codes for Curacao. Hence, the two data entries. But as of now, we were told that they
> have already updated the CSV generation process wherein the data is sorted by country code in order to avoid confusion.

Problem solved!

And I guess the best course of action now is probably to just manually edit the data based on what the CSV generates and restore the tests?

**Comment by raphaelvarieras** [comment]
Same issue it seems, this time [coming from Apple](https://github.com/fivetran/dbt_apple_store/issues/12).
